WBooks/Book Comments/BookCommentsViewModel.swift

```diff
@@ -7,12 +7,13 @@
 //
 
 import Foundation
+import ReactiveSwift
 
 class BookCommentsViewModel {
     
     var bookViewModel: BookViewModel
     
-    var comments: [Comment] = []
+    var comments = MutableProperty<[Comment]>([])
```

Comment:
The mutableProperty object does not changes, just it's value, so it could be `let`